pageserver/src/tenant/size.rs

```diff
@@ -85,23 +85,16 @@ pub struct TimelineInputs {
     #[serde_as(as = "Option<serde_with::DisplayFromStr>")]
     pub ancestor_id: Option<TimelineId>,
 
-    #[serde_as(as = "serde_with::DisplayFromStr")]
     ancestor_lsn: Lsn,
```

Comment:
I would prefer the usual Postgres formatting of LSNs. Is it a problem for the console?

Reply:
`Lsn` is `u64` which does not fully fit into JS, `f64`-ish number range (but JSON per se is fine with such numbers).
Could it potentially cause issues?

Reply:
`segments` array returns them in u64 and it's hard to match i.e. BranchStart with timeline gc_cutoff , when they are in different formats

We could change everything to X/X format, but it makes manual debugging of size calculation (lsn substraction) pretty inconvenient(